src/models/Guilds.ts

```diff
@@ -1,26 +1,18 @@
 import { model, Schema } from "mongoose";
 
-
 export interface GuildsI {
     guildId: string;
-    twitch: {
-        userId: string;
-    }
-    token: {
-        userToken: string;
-        refreshToken: string;
-    }
+    streams: { channelId: string; notify: boolean; message: string };
+    token: { userToken: string; refreshToken: string };
+    twitch: { userId: string };
 }
 
 const GuildsSchema = new Schema<GuildsI>({
     guildId: String,
-    twitch: {
-        userId: String,
-    },
-    token: {
-        userToken: String,
-        refreshToken: String
-    }
+    streams: { channelId: String, notify: Boolean, message: String },
```

Comment:
Add a default config for streams where `channelId` will be null `notify` will be false and `message` will be a default message to prevent a crash when any message was setup.